### PR TITLE
Default CA bundle name changed from 1.7 to 1.8

### PR DESCRIPTION
**Pull request by swizzley**
Tested on 1.8.2

Started PUT "/auth_source_ldaps/3******_" for 10.169.230.42 at 2015-08-07 17:55:12 +0000
2015-08-07 17:55:12 [I] Processing by AuthSourceLdapsController#update as HTML
2015-08-07 17:55:12 [I]   Parameters: {"utf8"=>"✓", "authenticity_token"=>"jn+gUw25uy4NNTUrNcfksXMy8idZkKjayG+0KuI7Fcw=", "auth_source_ldap"=>{"name"=>"LDAPS", "host"=>"myldaps.com", "tls"=>"1", "port"=>"3269", "server_type"=>"active_directory", "account"=>"ad\user", "account_password"=>"[FILTERED]", "base_dn"=>"DC=myldaps,DC=com", "groups_base"=>,DC=myldaps,DC=com", "ldap_filter"=>"memberOf=CN=Foremen_Svc_devOps,OU=Security Groups,OU=Corporate,DC=myldaps,DC=com", "onthefly_register"=>"1", "usergroup_sync"=>"0", "attr_login"=>"sAMAccountName", "attr_firstname"=>"givenName", "attr_lastname"=>"sn", "attr_mail"=>"mail", "attr_photo"=>""}, "id"=>"3-**_*******"}
2015-08-07 17:55:12 [I]

### CODE REVIEW

File: _includes/manuals/1.8/4.1.1_authentication.md [hunk 1]
```diff
@@ -33,6 +33,8 @@ On Red Hat based OSes:
 {% highlight bash %}
 cp example.crt /etc/pki/tls/certs/
 ln -s example.crt /etc/pki/tls/certs/$(openssl x509 -noout -hash -in /etc/pki/tls/certs/example.crt).0
+unlink /etc/pki/tls/certs/ca-bundle.crt
```
Comment (domcleal) [inline _includes/manuals/1.8/4.1.1_authentication.md:36]: I wouldn't recommend this, that's your system trust store and it might break other applications.

**Comment by swizzley** [comment]
No I don't think so, the global CA bundle is:  ca-bundle.trust.crt so relinking ca-bundle.crt has no adverse affects or risk. 

<pre>
 ll /etc/pki/tls/certs/
total 16
lrwxrwxrwx 1 root root   27 Aug  7 17:54 a******34.0 -> Root_CA.cer
lrwxrwxrwx 1 root root   10 Aug  7 17:54 ca-bundle.crt -> a******34.0
lrwxrwxrwx 1 root root   55 Jun 17 19:47 ca-bundle.trust.crt -> /etc/pki/ca-trust/extracted/openssl/ca-bundle.trust.crt
-rw------- 1 root root 1509 Jul 29 23:20 localhost.crt
-rwxr-xr-x 1 root root  610 Mar 23 21:04 make-dummy-cert
-rw-r--r-- 1 root root 2388 Mar 23 21:04 Makefile
-rwxr-xr-x 1 root root  829 Mar 23 21:04 renew-dummy-cert
</pre>

**Comment by domcleal** [comment]
ca-bundle.trust.crt is full of EV CA certificates, while ca-bundle.crt is the regular set, http://serverfault.com/questions/620003/difference-between-ca-bundle-crt-and-ca-bundle-trust-crt and https://access.redhat.com/solutions/476863 (if you have access).

I just tested this again on EL6.7 and it's working as I'd expect, running the standard instructions immediately fix my verification error.

Looking at your directory printout, is Root_CA.cer actually in /etc/pki/tls/certs/, or is it just missing from your ll output?

**Comment by mmoll** [comment]
I guess this can get closed?

**Comment by mmoll** [comment]
no reaction, closing.

**Pull request closed without merging**